.travis.yml

```diff
@@ -10,4 +10,4 @@ before_install:
   - go get github.com/mattn/goveralls
   - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
 script:
-    - $HOME/gopath/bin/goveralls -service=travis-ci -repotoken=$CODE_WALL_TOKEN
+    - $HOME/gopath/bin/goveralls -service=travis-ci -repotoken=COVERALLS
```

Comment:
is the token literally `COVERALLS`, or is that supposed to be a var like `$COVERALLS`?